dev/io.openliberty.org.jboss.resteasy.cdi.ee10/src/org/jboss/resteasy/cdi/ResteasyCdiExtension.java

```diff
@@ -51,299 +70,295 @@
  * Sesion Bean JAX-RS components.
  *
  * @author Jozef Hartinger
- *
  */
 public class ResteasyCdiExtension implements Extension {
```

Comment:
It will be.  In all likelihood I'll be merging the EE11 PR prior to this one.

Reply:
You updated the resteasy.cdi.ee11 bundle to 6.2.8 so you should also update the overlay as part of this PR?  Otherwise you should undo the bnd.bnd change and have both the 6.2.3 and 6.2.8 in the oss maven file.  Of course you will need to use the appropriate `exact` settings.